src/main/java/AddCommand.java

```diff
@@ -0,0 +1,20 @@
+public class AddCommand extends Command {
```

Comment:
I like how you created classes for each related code logics such as for commands to ensure single responsibility principle. However, would it be better if the class would be in a package? I noticed it with the other classes as well.